client/mapreduce/src/main/java/org/apache/accumulo/core/client/mapred/AbstractInputFormat.java

```diff
@@ -126,10 +125,13 @@ public static String getClassLoaderContext(JobConf job) {
    *          Connection information for Accumulo
    * @since 2.0.0
    */
-  public static void setConnectionInfo(JobConf job, ConnectionInfo info)
-      throws AccumuloSecurityException {
-    setConnectorInfo(job, info.getPrincipal(), info.getAuthenticationToken());
-    setZooKeeperInstance(job, ConnectionInfoFactory.getClientConfiguration(info));
+  public static void setConnectionInfo(JobConf job, ConnectionInfo info) {
+    ConnectionInfo inputInfo = InputConfigurator.updateToken(job.getCredentials(), info);
+    InputConfigurator.setConnectionInfo(CLASS, job, inputInfo);
+  }
+
+  protected static ConnectionInfo getConnectionInfo(JobConf job) {
```

Comment:
if its protected, then its in public API and should have a since tag

Reply:
Fixed now